samples/tools/testbed/run_scenarios.py

```diff
@@ -189,6 +189,7 @@ def run_scenario_in_docker(work_dir, timeout=600):
     with open(os.path.join(work_dir, "run.sh"), "wt") as f:
         f.write(
             """#
+umask 000
 . ./ENV
 pip install pyautogen
```

Comment:
This will install pyautogen<0.2.

Reply:
That it correct. In the current code it uses the current stable release. I've overridden this to test it on the 0.2.0 betas, and verified it will work when 0.2.0 goes to general availability (with the exception of the use_cache flag that I just added in this PR without remembering it was deprecated)